src/main/java/InputHandler.java

```diff
@@ -0,0 +1,127 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+
+/**
+ * Handles input in main in Duke.java. Receives String from Scanner in main
+ * Processes the input into 7 categories: Todo, Event, Deadline, list, mark, unmark, bye and throws error
+ */
+public class InputHandler {
+    private ArrayList<Task> arr;
```

Comment:
Might be better to have a more specific and plural name for a list of task.
Recommendation:
`
private ArrayList<Task> tasks;
`